collect_getherer.py

```diff
@@ -1,19 +1,70 @@
 #!/usr/bin/env python3
 
-# *******************************************************
-# * Copyright (c) 2022-2023 CAST.  All rights reserved. *
-# *******************************************************
-
 import os
+import argparse
+from argparse import Namespace
 import glob
 import shutil
+from typing import List, NoReturn
+
+CUR_DIR = os.path.dirname(os.path.abspath(__file__))
+COLLECTS_DIR = os.path.join(CUR_DIR, "collects")
+
+EXCLUDE_LIST = ["525.x264_r", "511.povray_r", "521.wrf_r",
```

Comment:
1. I suggest keeping this list in another dict where each item has the following form.
(bench_name: expected_runs_cunt)
2. EXCLUDE_LIST -> HAS_SIDE_BINARIES